public/javascripts/moneyTracker.js

```diff
@@ -64,7 +68,7 @@ app.run(function(editableOptions) {
                     $scope.addtoken=false;
                     $scope.subtracttoken=false;
                     $scope.eqtoken = false;
-                }else{
+                }else if($scope.output.length<10){
                     if(!(num == 0 && $scope.output == "-")){
                         $scope.output += String(num);
                     }
```

Comment:
What if it doesn't satisfy either else clause, we should probably throw an error message in that case